pages/404.tsx

```diff
@@ -0,0 +1,42 @@
+import { Flex, Heading, Text } from '@chakra-ui/react'
+import FlexBreak from '../components/common/FlexBreak'
+import SimpleLink from '../components/common/SimpleLink'
+
+interface ErrorPageProps {
+  code: number
+  message: string
+}
+
+/**
+ * A reusable error page: it automatically handles Next errors, but it can also be used with a custom status code and message.
+ * @param code the status code referred to the error, defaults to `404` for Next automatic error handling.
+ * @param message the message to display with the error, defaults to "Not found" for Next automatic error handling.
+ */
+export default function ErrorPage({
+  code = 404,
+  message = 'Not found',
+}: ErrorPageProps): JSX.Element {
+  return (
+    <Flex flexWrap="wrap" justifyContent="center" alignItems="center" height="100%">
+      <Flex flexWrap="wrap" justifyContent="center">
+        <Heading>Ooops... there was an error!</Heading>
+
+        <FlexBreak></FlexBreak>
```

Comment:
I don't think you would need a FlexBreak component if you used [flexDirection="column"](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#flex-direction)? Another option would be to wrap the content that you want centred in one element and use that as the child of a [Center](https://chakra-ui.com/docs/layout/center).

Reply:
Yep! You're right, that was indeed a very ugly solution 😅. Fixed in the latest commit.